webapp/locales/es.json

```diff
@@ -119,6 +173,113 @@
     "tag": "Etiqueta ::: Etiquetas",
     "name": "Nombre",
     "loadMore": "cargar más",
-    "loading": "cargando"
+    "loading": "cargando",
+    "reportContent": "Report"
+  },
+  "actions": {
+    "loading": "cargamento",
+    "loadMore": "cargar más",
+    "create": "Crear",
+    "save": "Guardar",
+    "edit": "Edite",
+    "delete": "Delete",
+    "cancel": "Cancelar"
+  },
+  "moderation": {
+    "name": "Moderación",
+    "reports": {
+      "empty": "Felicitaciones, nada que moderar.",
+      "name": "Informes",
+      "submitter": "comunicado por",
+      "disabledBy": "desactivado por"
+    }
+  },
+  "disable": {
+    "submit": "Desactivar",
+    "cancel": "Cancelar",
+    "success": "Discapacitado con éxito",
+    "user": {
+      "title": "Desactivar usuario",
+      "type": "Usuario",
+      "message": "¿Realmente quieres deshabilitar el usuario \"<b>{name}</b>\"?"
+    },
+    "contribution": {
+      "title": "Deshabilitar contribución",
+      "type": "Contribución",
+      "message": "¿Realmente quieres deshabilitar la contribución \"<b>{name}</b>\"?"
```

Comment:
Whaaaat, this is possible? I didn't know you can add variables to a translation.

Reply:
ahhh ... there is a mistake ... the B element closes but it does not open ... 
an error which e.g. is not recognized!

Reply:
what do you mean @ogerly... I needed to add this `<b></b>` to a `v-html` for it to get converted to bold, instead of it being read literally, but maybe that is not what you mean at all.

@roschaefer, I was also pleasantly surprised to see this!! and it's super handy when you need to pass in computed values, like the `commentsCount`, `contributionsCount`